### PR TITLE
fish: fix typo, omf: add page

### PR DESCRIPTION
**Pull request by TempPlaceHolder**
omf: add page

### CODE REVIEW

File: pages/common/omf.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# omf
+
+> Oh My Fish, the Fishshell Framework.
+> Install packages to extend and modify the fish shell.
+
+- Install one or more packages:
```
Comment (agnivade) [inline pages/common/omf.md]: How are multiple packages specified ? Separated by space ? We should mention that.

File: pages/common/omf.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- List available themes:
+
+`omf theme`
+
+- Apply a theme:
+
+`omf theme {{name}}`
+
+- Remove a theme or package:
```
Comment (agnivade) [inline pages/common/omf.md]: What if there is a theme and package with the same name ? Both will be removed ?

File: pages/common/omf.md (hunk 1 above)
Comment (pxgamer) [reply pages/common/omf.md]: Yeh, they are space separated according to the docs. :+1:

File: pages/common/omf.md (hunk 2 above)
Comment (TempPlaceHolder) [reply pages/common/omf.md]: Hm themes are also packages, so it doesn't matter.

Comment (agnivade) [reply pages/common/omf.md]: I see. Ok.

**Pull request merged**